src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,81 @@
+package duke;
+
+import duke.exception.DukeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Storage {
+
+    public static final String line = "//";
```

Comment:
Should the naming for line be in uppercase since it is a constant? I noticed the same issue in several other files too.